packages/remix-react/links.ts

```diff
@@ -221,12 +221,42 @@ export function getLinksForMatches(
   return dedupe(descriptors, preloads);
 }
 
+let stylesheetPreloadTimeouts = 0;
+let isPreloadDisabled = false;
+
 export async function prefetchStyleLinks(
   routeModule: RouteModule
 ): Promise<void> {
   if (!routeModule.links) return;
   let descriptors = routeModule.links();
   if (!descriptors) return;
+  if (isPreloadDisabled) return;
+
+  // If we've hit our timeout 3 times, we may be in firefox with the
+  // `network.preload` config disabled and we'll _never_ get onload/onerror
+  // callbacks.  Let's try to confirm this with a totally invalid link preload
+  // which should immediately throw the onerror
+  if (stylesheetPreloadTimeouts >= 3) {
```

Comment:
Commented on the Firefox bug, but you really want `document.createElement("link").relList.supports("preload")` or so.

In any case, I wouldn't bother dealing with it unless you want to also deal with browsers that don't support `<link rel=preload>` (which is fine, but can be simpler as per the above).